include/openssl/ec.h

```diff
@@ -269,6 +269,23 @@ OPENSSL_EXPORT int EC_POINT_point2cbb(CBB *out, const EC_GROUP *group,
                                       point_conversion_form_t form,
                                       BN_CTX *ctx);
 
+// EC_POINT_point2buf allocates a buffer of suitable length and writes an
+// EC_POINT to it in octet format. The allocated buffer is written to *pbuf and
+// its length is returned. The caller must free up the allocated buffer with a
+// call to OPENSSL_free(). Since the allocated buffer value is written to *pbuf
+// the pbuf parameter MUST NOT be NULL.
+OPENSSL_EXPORT size_t EC_POINT_point2buf(const EC_GROUP *group,
```

Comment:
Outside the scope for this PR, but for this to be useful do we need a `EC_POINT_buf2point`? What would someone do with this serialized buffer?

Reply:
I agree, there's no need to expose this function. I'll change it to be a `static` function.